src/main/java/seedu/address/ui/CompletableTodoCard.java

```diff
@@ -30,6 +32,14 @@ public CompletableTodoCard(CompletableTodo todo, int displayedIndex) {
         this.todo = todo;
         id.setText(displayedIndex + ". ");
         description.setText(todo.getDescription());
+        completedLabel.setText(todo.getIsDone() ? "✔" : "");
```

Comment:
Do you wanna simplify "todo.getIsDone() ? "✔" : """ with a method in Todo? Or even from toString?